### ISSUE TITLE
Feature request: Option to make loop timer match current time

### ISSUE THREAD

**Issue by ewstr** [dziemborowicz/hourglass#203]
I like the app very useful,

Could you add a feature where you can start any time loop lets say for example 15mins, if you start that at  9:51am it will run until 10:00am then it will beep the next one will beep at 10:15am and the next will beep at 10:30am and so on always matching the current time. something like the iPhone app called chime made by Blacktree.

Thank you in advance.

**Comment by Seb90258** [comment]
Yeah, I vote for this. I like to use the this as a floating timer synced to TradingView/Binance and I guess this means having it synced to match the current time.
